react/components/Table/SimpleTable.js

```diff
@@ -284,8 +284,16 @@ class SimpleTable extends Component {
                           schema.properties[property].title || property
                         const headerRight =
                           schema.properties[property].headerRight || false
-                        const headerRenderer =
-                          schema.properties[property].headerRenderer
+                        const header =
```

Comment:
Did you test this new option with all available possibilities?

Reply:
Just tested [here](https://test--sandboxintegracao.myvtex.com/admin/received-skus/pending/?elements=15&from=0&sortOrder=ASC&sortedBy=stock&to=15) and got a bug. Pushing fix